README.md

```diff
@@ -10,8 +10,8 @@ ECMAScript Stage-0 Proposal. J. S. Choi, 2021.
 (A [formal specification][] is available.)
 
 **Method binding** `->` is a **left-associative infix operator**.
-Its right-hand side is an **identifier**
-or an **expression** in `(` `)`,
+Its right-hand side is a **chain of identifiers identifier** (like `f` or `o.x.y`)
+or an **expression** in `(` `)` (like `hof()`),
```

Comment:
the example is a function invocation, not an expression in parens

Reply:
Ah, this is an inconsistency. It should just be “identifier”; no `o.x.y` allowed.

I had been planning to allow method chains on the RHS (see https://github.com/js-choi/proposal-bind-this-operator/issues/3#issuecomment-920563136), but I decided against this for now to keep precedence simple. I am open to properly changing the RHS to allow property chains later.